README.md

```diff
@@ -8,64 +8,39 @@
 
-Terra Toolkit is a utility module used to facilitate independent development of Terra projects. This toolkit provides build scripts, configurations, and Webdriver Services needed to serve assets, compile webpack, and run webdriver.io tests to streamline development of npm packages.
+Terra Toolkit is a [Lerna](https://github.com/lerna/lerna) repository for common development packages necessary for developing Terra projects.
 
-[terra-core][@terra-core], [terra-clinical][@terra-clinical], and [terra-framework][@terra-framework] are a few examples repositories which are utilizing the utilities offered in this package.
-
-[terra-dev-site][@terra-dev-site] is a node module that extends the configurations offered by toolkit.
-
-- [Getting Started](#getting-started)
-- [Serve Options](#serve-options)
-- [Webdriver.io Utility](#webdriverio-utility)
-- [Webpack Configuration](#webpack-configuration)
+- [Notice](#notice)
+- [Package Status](#package-status)
 - [Versioning](#versioning)
 - [Contributing](#contributing)
 - [LICENSE](#license)
 
-## Getting Started
-
-- Install toolkit and its peer dependencies as dev-dependencies with [npm](https://www.npmjs.com): 
-```bash
-> npm install --save-dev terra-toolkit webpack webpack-cli webpack-dev-server @babel/cli @babel/core 
-```
-
-To use Terra Toolkit for local webdriver testing, you must install docker on your machine.
-
-- Install Docker version 17.09.0 or higher. Installation instructions can be found at https://docs.docker.com/install/.
-
-## Theme Aggregation
-
-Terra Toolkit provides a built-in mechanism for aggregating themes.
-
-See the [Theme Aggregation](https://github.com/cerner/terra-toolkit/blob/master/scripts/aggregate-themes/README.md) to get started.
-
-## Serve Options
-
-Terra Toolkit offers two ways to serve your client side application, `serve` and `serve-static`. Serve provides a hot-reloading, development only option via [webpack-dev-server](https://github.com/webpack/webpack-dev-server), while serve-static is a non-hot-reloading [express](https://expressjs.com/) server. Serve-static supports IE10 and is used in webdriver testing.
-
-See the [Serve Guide](https://github.com/cerner/terra-toolkit/blob/master/scripts/serve/Serve.md) to get started.
-
-## Webdriver.io Utility
-
-[Webdriver.io](http://v4.webdriver.io/) is a framework for writing webdriver powered tests to validate functionality in browsers. The Webdriver.io framework provides services for setting up a selenium server, starting webpack and static servers, running accessibility and visual regression testing, and more.
+## Notice
 
-See the [Webdriver.io Utility Developer's Guide](https://github.com/cerner/terra-toolkit/blob/master/docs/Wdio_Utility.md) to get started.
+The code for the [terra-toolkit](https://www.npmjs.com/package/terra-toolkit) npm dependency has been moved to [terra-toolkit-boneyard](https://github.com/cerner/terra-toolkit-boneyard).
```

Comment:
can we note that this code is currently in development?

```suggestion
This code is currently in development. The code for the [terra-toolkit](https://www.npmjs.com/package/terra-toolkit) npm dependency has been moved to [terra-toolkit-boneyard](https://github.com/cerner/terra-toolkit-boneyard).
```

Reply:
What is `this` in this context? Are you wanting to add a note that terra-toolkit package code is still being developed in terra-toolkit-boneyard? Or that terra-toolkit in this repository is under development?

Reply:
the terra-toolkit in this repo

Reply:
Can you provide a few more details about adding the development notes? There are two packages in this repo that are released artifacts on npm. Generally all repository code falls under "in development". Are you wanting to clarify that the terra-functional-testing package specifically is under development and not yet released?

Reply:
ah, yes, unreleased is a better term. not a huge deal tho

Reply:
I think I'll leave it as is for now. The browserslist-config-terra and eslint-config-terra are stable releases from this repository. Only the terra-functional-testing package is unreleased right now so the note doesn't apply to everything. We can circle back around if we want to update the README with additional information.  For now I'd like to get this merged to help guide people to terra-toolkit-boneyard if that is what they are looking for.